### DISCUSSION TITLE
popup / suggestions - disable

### DISCUSSION THREAD

**Discussion by aum7** [LunarVim/LunarVim#3807] [Q&A] [answered]
olo
i want to disable popups / suggestions / completions,
completely

how can i disable popup windows : 
- temporarily, ie for a lvim session ?
- permanently, ie when lvim starts, this behavior is not loaded ?

godspeed

aum
s love nia
![lvim_popups](https://user-images.githubusercontent.com/9611505/216575935-4979c51f-8a92-445e-8c90-399a07e0012a.png)

**Comment by LostInTheLogs** [comment, marked as answer]
disable the cmp plugin, you can also add custom logic to its enable function if you want greater control

**Reply by aum7** [reply]
thank you for your reply
can you point me to more info, how to add custom logic to it, if i want greater control ?

have fun

**Reply by LostInTheLogs** [reply]
here's the default function, modify it to meet your needs
```lua
lvim.builtin.cmp.enabled = function()
  local buftype = vim.api.nvim_buf_get_option(0, "buftype")
  if buftype == "prompt" then
    return false
  end
  return lvim.builtin.cmp.active
end,
```

**Reply by aum7** [reply]
thank you for your proper reply,
appreciated

have fun
aum
